src/app/locale/translations/en.json

```diff
@@ -197,7 +197,11 @@
             "checkoutcom_document_invalid_error_ideal": "BIC must be 8 or 11 characters",
             "checkoutcom_document_invalid_error_oxxo": "CURP must be at most 18 characters",
             "checkoutcom_document_invalid_error_qpay": "National ID must be at most 32 characters",
-            "checkoutcom_sepa_mandate_disclaimer":"By accepting this mandate form, you authorize {storeName} and Checkout.com, our payment service provider, to send instructions to your bank to debit your account, and your bank to debit your account in accordance with those instructions. You are entitled to a refund from your bank under the terms and conditions of your agreement with your bank. A refund must be claimed within 8 weeks starting from the date on which your account was debited.",
+            "checkoutcom_fawry_customer_mobile_invalid_error": "A valid Mobile Number is required",
+            "checkoutcom_fawry_customer_mobile_label": "Mobile Number",
+            "checkoutcom_fawry_customer_email_invalid_error": "Email address must be valid",
+            "checkoutcom_fawry_customer_email_label": "Email",
+            "checkoutcom_sepa_mandate_disclaimer": "By accepting this mandate form, you authorize {storeName} and Checkout.com, our payment service provider, to send instructions to your bank to debit your account, and your bank to debit your account in accordance with those instructions. You are entitled to a refund from your bank under the terms and conditions of your agreement with your bank. A refund must be claimed within 8 weeks starting from the date on which your account was debited.",
```

Comment:
where is this being used?

Reply:
it's used in `src/app/payment/paymentMethod/CheckoutcomCustomFormFields.tsx`, I modified it it to alphabetize the section and because it was missing the space after `:`